examples/basics/main.rs

```diff
@@ -22,9 +22,7 @@ fn main() {
         gtk::Application::new(Some("com.github.gtk-rs.examples.basic"), Default::default())
             .expect("Initialization failed...");
 
-    application.connect_activate(|app| {
-        build_ui(app);
-    });
+    application.connect_activate(build_ui);
```

Comment:
Interesting, why didn't clippy warning about that?

Reply:
Good question